cyberbrain/tracer.py

```diff
@@ -29,6 +29,24 @@
     _debug_mode = cb_args.debug_mode
 
 
+class TracebackHandler:
```

Comment:
I would just use a function, something like `get_function_lineno_from_tracer`.

A class seems overkill. We don't need to store `self._function_lineno`, and can just return it when the function gets called. It will only be called once for each frame, so `self._active` is also unnecessary.

Reply:
I changed the logic to drop the class. You were right about it being overkill. I wanted a way to manage the state being carried over. I've pushed a new solution. Hopefully it's better aligned with what you had in mind.